src/components/edge.js

```diff
@@ -679,7 +678,8 @@ class Edge extends React.Component<IEdgeProps> {
             xlinkHref={Edge.getXlinkHref(edgeTypes, data)}
             width={edgeHandleSize}
             height={edgeHandleSize}
-            transform={`${this.getEdgeHandleTransformation()}`}
+            transform={edgeHandleTransformation}
+            style={{ transform: edgeHandleTransformation }}
```

Comment:
This style attribute fixes a bug with either Chrome or d3 randomly adding a `transform: matrix` style.